lib/storage-service.js

```diff
@@ -53,6 +53,29 @@ function StorageService(options) {
   if (options.maxFieldsSize) {
     this.maxFieldsSize = options.maxFieldsSize;
   }
+  // pass through AWS specific options
```

Comment:
"AWS specific options" should suffice, IMO.

Reply:
I've changed this in the latest commit